app/Main.hs

```diff
@@ -126,19 +128,19 @@ main = do
       setupNixpkgs token
       P.setEnv "GITHUB_TOKEN" (T.unpack token) True
       deleteDone token
-    UpdateList UpdateOptions {pr, cachix, cve, outpaths} -> do
+    UpdateList UpdateOptions {pr, cachix, cve, nixpkgsReview, outpaths} -> do
       token <- getGithubToken
       updates <- T.readFile "packages-to-update.txt"
       setupNixpkgs token
       P.setEnv "PAGER" "" True
       P.setEnv "GITHUB_TOKEN" (T.unpack token) True
-      updateAll (Options pr True token cve cachix outpaths) updates
-    Update UpdateOptions {pr, cve, cachix} update -> do
+      updateAll (Options pr True token cve cachix nixpkgsReview outpaths) updates
+    Update UpdateOptions {pr, cve, cachix, nixpkgsReview} update -> do
       token <- getGithubToken
       setupNixpkgs token
       P.setEnv "PAGER" "" True
       P.setEnv "GITHUB_TOKEN" (T.unpack token) True
-      result <- updatePackage (Options pr False token cve cachix False) update
+      result <- updatePackage (Options pr False token cve cachix nixpkgsReview False) update
```

Comment:
I vaguely recall there's a Haskell language extension that allows you to do something like name locally scoped variables with the same names as the `Options` fields, and then just initialize one by doing `Options(..)`.  Might want to consider for all this repetition (assuming that is how it behaves, been a while).

Reply:
Yeah, the Options thing is becoming a mess! I want to unify the options in Main with the options in Utils, but haven't gotten around to it yet.